### DISCUSSION TITLE
Calculate moon face orientation as seen from point on earth

### DISCUSSION THREAD

**Discussion by bjarkeandersen** [cosinekitty/astronomy#367] [Q&A] [answered]
I would like to know how to calculate the orientation of the moon face as seen from a point on earth. I already have calculated the orientation of the shadow on the moon, but since we use a static image of the moon (https://www.yr.no/assets/images/moon.png) it would be nice to rotate the image to make it look almost identical to reality. Anyone who knows how to find the degrees of orientation?

**Comment by cosinekitty** [comment, marked as answer]
Take a look in the demo programs for whichever programming language you are using. There is one called "camera" that does exactly this. For extra realism, use the libration functions to determine the slight shifting of the side facing the earth.

**Reply by bjarkeandersen** [reply]
Of course - the moon rotates the same as the earth shadow on the moon depending on your viewpoint on earth. The shadow on the moon stays the same no matter where you see it from on earth. Some times it takes some time before I manage to picture this in my head. Thank you so much for a quick answer.
